cdhweb/blog/models.py

```diff
@@ -99,4 +107,96 @@ def get_absolute_url(self):
             # force two-digit month
             'month': '%02d' % self.publish_date.month,
             'day': '%02d' % self.publish_date.day,
-            'slug': self.slug})
\ No newline at end of file
+            'slug': self.slug})
+
+
+# custom manager for wagtail pages, see:
+# https://docs.wagtail.io/en/stable/topics/pages.html#custom-page-managers
+BlogPostManager = PageManager.from_queryset(BlogPostQuerySet)
+
+
+class BlogPostTag(TaggedItemBase):
+    """Tags for Blog posts."""
+    content_object = ParentalKey(
+        "blog.BlogPost", on_delete=models.CASCADE, related_name="tagged_items")
+
+
+class BlogPost(Page, ClusterableModel):
+    """A Blog post, implemented as a Wagtail page."""
+
+    authors = models.ManyToManyField("people.Person", related_name="posts")
```

Comment:
Can we easily make author an ordered many to many with wagtail? Probably should be out of scope if it's not easy, but thought I would ask. (It's something we've wanted before)

Reply:
Great q! I know wagtail has an `Ordered` mixin and this seems like a core use-case; maybe they have an example already. I'll look